src/app/components/drawer/base-drawer.tsx

```diff
@@ -56,20 +55,19 @@ export const BaseDrawer = memo((props: BaseDrawerProps) => {
   } = props;
   const ref = useDrawer(isShowing, onClose ? onClose : noop, pauseOnClickOutside);
   const navigate = useNavigate();
-  const { theme } = useThemeSwitcher();
 
   const onGoBack = () => navigate(-1);
 
   return (
     <Flex
       display={isShowing ? 'flex' : 'none'}
-      bg={`rgba(0,0,0,0.${isShowing ? 4 : 0})`}
+      bg="overlay"
```

Comment:
Removed conditionality here?

Reply:
I don't think this was ever nec bc of the display conditional above it. It seemed fine to remove it.

Reply:
For stuff like this I think we should remove it as you did, then re-add it if it's noticed as a bug. That will help us simplify things as we go too